BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.java

```diff
@@ -130,6 +129,8 @@ private UUID scheduleAnalyticsUpload(Configuration configuration, Authorization
 
     ListenableWorker.Result uploadAnalytics(Context context, Data inputData) {
         Configuration configuration = getConfigurationFromData(inputData);
+        Authorization authorization = getAuthorizationFromData(inputData);
```

Comment:
Do we need to add a null check for this on line 137?

Reply:
I guess technically we could in order to make the linter happy, but we do produce the `inputData` being sent in here.

Reply:
It is a good call here to defensively check for `null`. I'll add some tests in for that too.

Reply:
Added the `null` checks!